lib/fileutils.rb

```diff
@@ -170,12 +195,32 @@ def remove_trailing_slash(dir)   #:nodoc:
   private_module_function :remove_trailing_slash
 
   #
-  # Creates one or more directories.
+  # Creates directories at the paths in the given +list+ (an array of strings);
+  # returns +list+.
```

Comment:
I think we should mention that `list` could also be a string (to create just a single directory).

Reply:
Done.